ui/src/app/base/components/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { shallow } from "enzyme";
+import React from "react";
+
+import Slider from "./Slider";
+
+describe("Slider", () => {
+  it("renders without number input", () => {
+    const wrapper = shallow(
+      <Slider max={10} min={0} onChange={jest.fn()} value={5} />
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("renders with number input", () => {
+    const wrapper = shallow(
+      <Slider max={10} min={0} onChange={jest.fn()} showInput value={5} />
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
I might be good to be explicit in this or the above test so it's clear what's being tested.

Reply:
Good call